Client/Frontend/Fakespot/FakespotViewModel.swift

```diff
@@ -37,10 +37,14 @@ class FakespotViewModel {
         actionTitle: .Shopping.ErrorCardButtonText
     )
 
-    let highlightsCardViewModel = HighlightsCardViewModel(
-        footerTitle: .Shopping.HighlightsCardFooterText,
-        footerActionTitle: .Shopping.HighlightsCardFooterButtonText
-    )
+    let highlightsCardViewModel = {
```

Comment:
Are these non-localized strings just placeholders?

Reply:
Oh right, will add a comment. It's just dummy data to show something until we integrate with the API.